package.json

```diff
@@ -55,6 +55,8 @@
     "test:e2e:bootc": "npm run test:e2e:copy && npm run test:e2e:build && npm run test:e2e:bootc:run",
     "test:e2e:bootc:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- vitest run tests/src/bootc-extension.spec.ts --pool=threads --poolOptions.threads.singleThread --poolOptions.threads.isolate --no-file-parallelism",
     "test:e2e:copy": "cp ../podman-desktop-extension-bootc/tests/src/bootc-extension.spec.ts tests/src/",
+    "test:e2e:pmachine": "npm run test:e2e:build && npm run test:e2e:pmachine:run",
+    "test:e2e:pmachine:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- vitest run tests/src/podman-machine-dashboard-onboarding.spec.ts --pool=threads --poolOptions.threads.singleThread --poolOptions.threads.isolate --no-file-parallelism",
```

Comment:
not clear to me why we need again new entries there ?

Reply:
if you need some selector I think it might be better to have a generic target where you can provide the specific path to the file

If we need a script for each file I think it's not scaling

Reply:
@benoitf I agree, this solution is not scaling. 
@cbr7 you are also right, it is hard to include test that messes(tests) with underlying env. and influences other tests. 
In other tests (podman onboarding where PR is opened) @ScrewTSW added a env. vars. to operate the tests, I think this is also something we could make use of. 
I am working on a solution to propagate env. vars, from GHA workflows inputs into testing images on gha.

Reply:
@odockal which PR is this one that you mentioned so I can take a look and use the same env var.

Reply:
Nevermind found the PR.